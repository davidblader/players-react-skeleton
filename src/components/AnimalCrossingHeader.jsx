import React from 'react';
import PropTypes from 'prop-types';

const AnimalCrossingHeader = props => (<span className="animal-crossing-header">{props.children}</span>);

AnimalCrossingHeader.propTypes = {
  children: PropTypes.node.isRequired,
};

export default AnimalCrossingHeader;
