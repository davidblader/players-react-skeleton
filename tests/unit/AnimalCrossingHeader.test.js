import React from 'react';
import { shallow } from 'enzyme';
import AnimalCrossingHeader from '../../src/components/AnimalCrossingHeader';

test('AnimalCrossingContainer shallow renders', () => {
  const component = shallow(<AnimalCrossingHeader>Test</AnimalCrossingHeader>);
  expect(component).toMatchSnapshot();
});
