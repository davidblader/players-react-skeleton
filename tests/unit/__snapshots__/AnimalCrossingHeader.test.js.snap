// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnimalCrossingContainer shallow renders 1`] = `
<span
  className="animal-crossing-header"
>
  Test
</span>
`;
